server/core/terraform/terraform_client_test.go

```diff
@@ -380,21 +380,48 @@ terraform {
 	// version of 1.3.5.
 	// nonExactSymbols := []string{">", ">=", "<", "<=", "~>"}
 	nonExactSymbols := []string{"<", "<=", "~>"}
-	nonExactVersions := map[string]map[string]string{
-		// ">": {
-		// 	"project1": "1.3.5",
-		// },
-		// ">=": {
-		// 	"project1": "1.3.5",
-		// },
-		"<": {
-			"project1": "0.12.7",
+	expectedVersions := map[string]map[string]map[string]string{
+		"0.12.8": {
+			"=": {
+				"project1": "0.12.8",
+			},
```

Comment:
Few things

- This is a bit hard to read because usually, I would read something like this `= 0.12.8` would return version `0.12.8`

- The `project1` key doesn't change at all.

Could we have a structure like this instead?

```go
	{
		// left side is `required_version` and
		// right side is the downloaded terraform version
		"= 0.12.8": "0.12.8",
		"< 0.12.8": "0.12.7",
		"<= 0.12.8": "0.12.31",
		"~> 0.12.8": "0.12.31",
	}
```

Reply:
yeah sure, let me change it